modules/core/src/main/scala/com.snowplowanalytics.snowplow.snowflake/processing/SnowflakeRetrying.scala

```diff
@@ -35,15 +35,16 @@ object SnowflakeRetrying {
 
   /** Is an error associated with setting up Snowflake as a destination */
   private def isSetupError: PartialFunction[Throwable, String] = {
+    case ire: IngestResponseException if ire.getErrorCode === 513 || ire.getErrorCode === 404 =>
```

Comment:
I left `513` in case it can be returned somehow even though I can't trigger it. Should we remove it ?